app/models/finance/billing_strategy.rb

```diff
@@ -15,26 +15,15 @@ class << self
 
   attr_reader :failed_buyers
 
-  CURRENCIES = {
-    'USD - American Dollar' => 'USD',
-    'EUR - Euro'=> 'EUR',
-    'GBP - British Pound' => 'GBP',
-    'NZD - New Zealand dollar' => 'NZD',
-    'CNY - Chinese Yuan Renminbi' => 'CNY',
-    'CAD - Canadian Dollar' => 'CAD',
-    'AUD - Australian Dollar' => 'AUD',
-    'JPY - Japanese Yen' => 'JPY',
-    'CHF - Swiss Franc' => 'CHF',
-    'SAR - Saudi Riyal' => 'SAR'
-  }.freeze
+  CURRENCIES = CurrenciesLoader.load_config
```

Comment:
Maybe freeze it? So it cannot be modified on runtime

Reply:
Right, done 😎